drivers/include/at86rf2xx.h

```diff
@@ -106,57 +108,54 @@ extern "C" {
 
 /**
  * @brief   Internal device option flags
+ *
+ * `0x00ff` is reserved for general IEEE 802.15.4 flags
+ * (see @ref netdev2_ieee802154_t)
+ *
  * @{
  */
-#define AT86RF2XX_OPT_AUTOACK        (0x0001)       /**< auto ACKs active */
-#define AT86RF2XX_OPT_CSMA           (0x0002)       /**< CSMA active */
-#define AT86RF2XX_OPT_PROMISCUOUS    (0x0004)       /**< promiscuous mode
+#define AT86RF2XX_OPT_SRC_ADDR_LONG  (NETDEV2_IEEE802154_SRC_MODE_LONG) /**< legacy define */
+#define AT86RF2XX_OPT_RAWDUMP        (NETDEV2_IEEE802154_RAW)           /**< legacy define */
+#define AT86RF2XX_OPT_AUTOACK        (NETDEV2_IEEE802154_ACK_REQ)       /**< legacy define */
+
+#define AT86RF2XX_OPT_CSMA           (0x0100)       /**< CSMA active */
+#define AT86RF2XX_OPT_PROMISCUOUS    (0x0200)       /**< promiscuous mode
                                                      *   active */
-#define AT86RF2XX_OPT_PRELOADING     (0x0008)       /**< preloading enabled */
-#define AT86RF2XX_OPT_TELL_TX_START  (0x0010)       /**< notify MAC layer on TX
+#define AT86RF2XX_OPT_PRELOADING     (0x0400)       /**< preloading enabled */
+#define AT86RF2XX_OPT_TELL_TX_START  (0x0800)       /**< notify MAC layer on TX
```

Comment:
Short answer: read the doc above. Long answer: Because `0x0008` and `0x0010` are already used by the `netdev2_ieee802154` flags.